jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/GridListTests.java

```diff
@@ -0,0 +1,54 @@
+package io.github.epam.angular.tests.elements.complex;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.gridListSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class GridListTests extends TestsInit {
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+    }
+
+    @Test
+    public void basicGridListBasicTest() {
+        gridListSection.basicGridList.is().displayed();
+        gridListSection.basicGridList.get(1).show();
+    }
+
+    @Test
+    public void basicGridListTextTest() {
+        gridListSection.basicGridList.get(1).is().text("1");
+    }
+
+    @Test
+    public void basicGridListColorTest() {
+        gridListSection.dynamicGridList.get(1)
```

Comment:
It looks like a typo: gridListSection.dynamicGridList. Probably meant gridListSection.bascisGridList?